dynamic-layers/meta-arm/recipes-bsp/trusted-firmware-a/trusted-firmware-a_%.bbappend

```diff
@@ -1,6 +1,6 @@
 # Common Build targets
-TFA_BUILD_TARGET = "all"
-TFA_INSTALL_TARGET = "bl31"
+TFA_BUILD_TARGET ?= "all"
+TFA_INSTALL_TARGET ?= "bl31"
```

Comment:
I agree with you, but I would like to suggest overriding it only for the machines supported on the `bbappend`.